src/methods.ts

```diff
@@ -110,6 +113,55 @@ class JsonRpcMethods {
             })
             .catch((e) => nodeError(e, callback));
     }
+
+    getInstanceInfo(
+        blockHash: string,
+        index: bigint | number | string,
```

Comment:
What is the reason for allowing multiple types here? I think that will just cause confusion.